package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "ecommerce-app-legends-bn",
+  "version": "1.0.0",
+  "description": "## Introduction",
+  "main": "index.js",
+  "scripts": {
+    "build": "babel src/index.js -d dist",
+    "start": "npm run build && nodemon dist/index.js",
```

Comment:
- remove the build command in the start script. 
-  also don't use nodemon to run the built app. Just use the good old node command.